### PR TITLE
fix #60821 ani24.do.com

### LINKED ISSUE #60821: ani24do.com
### Issue URL (Anti Adblock Script)
 [https://ani24do.com/ani/search.php?type=all&order=view](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fani24do.com%2Fani%2Fsearch.php%3Ftype%3Dall%26order%3Dview)
### Comment
Username: @nam918
> .

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese, <br/>EasyList China, <br/>EasyList Czech and Slovak, <br/>List-KR, <br/>YousList

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#60821

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -5214,6 +5214,9 @@ dap-news.com##.ads
 ! https://github.com/AdguardTeam/AdguardFilters/issues/60583
 !+ PLATFORM(windows, mac, android)
 @@||cmt.dcnewsj.joins.com//web//reply.css$domain=dcnewsj.joins.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/60821
+ani24do.com#%#AG_onLoad(function(){document.querySelector("#box a[href^='/ads.php?ads=']")&&setTimeout(function(){var a=document.querySelector('#box');a&&a.click()},1500)});
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:5218]: better `document.querySelector('#box a[href^="/ads.php?ads="]')`

Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt:5218]: Btw I cannot find where it must be applied

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/foreign.txt:5218]: Screenshot 2020-08-05 at 19 25 32
Колбек обработчика кликов
Screenshot 2020-08-05 at 19 27 46

Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt:5218]: what about the first message?

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/foreign.txt:5218]: So no another remarks? Ok

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**